client.go

```diff
@@ -5,17 +5,18 @@ import (
 	"io"
 	"sync"
 
-	stats "github.com/libp2p/go-libp2p-gorpc/stats"
 	host "github.com/libp2p/go-libp2p-host"
 	inet "github.com/libp2p/go-libp2p-net"
 	peer "github.com/libp2p/go-libp2p-peer"
 	protocol "github.com/libp2p/go-libp2p-protocol"
+
+	stats "github.com/libp2p/go-libp2p-gorpc/stats"
```

Comment:
not sure why this moved

Reply:
something that my ide does on its own, go-imports I guess